pkg/sql/sqlinstance/instancestorage/instancereader_test.go

```diff
@@ -247,19 +283,13 @@ func TestReader(t *testing.T) {
 				t.Fatal(err)
 			}
 			testutils.SucceedsSoon(t, func() error {
-				instances, err := reader.GetAllInstances(ctx)
-				if err != nil {
-					return err
-				}
-				sortInstances(instances)
-				return testOutputFn(
+				return verifyInstances(t, expectations{
```

Comment:
I'd expect the linter to want field names, generally our style is to write them

Reply:
I don't think we lint against that although it'd be nice. I didn't add them initially because I assumed the types would be enough.